src/UserStories/UserStories.css

```diff
@@ -124,10 +124,13 @@ html {
 }
 .story-body {
   transition: transform 0.5s ease-out;
+  border-radius: 8px;
+  overflow: hidden;
```

Comment:
Let's let this one slip this time, but this is called "[scope creep](https://en.wikipedia.org/wiki/Scope_creep)". Which means sneaking things that are not related to the PR in the PR. 

These changes are good, but when they sneak into a PR meant for something else, they become undocumented. Because when you see the commit history later, you'll see "Fix avatar colored circle", but these changes have nothing to do with that. 

You can keep them this time but try to make your PR include only the changes it says it includes 👍🏼